src/lib/registration/sw_mgmt.rb

```diff
@@ -443,18 +444,22 @@ def self.copy_old_credentials(source_dir)
         ::FileUtils.mkdir_p(dir)
       end
 
-      # check for NCC credentials
-      ncc_file = File.join(source_dir, dir, "NCCcredentials")
-      copy_old_credentials_file(ncc_file)
+      Dir[File.join(source_dir, dir, "*")].each do |path|
+        # skip non-files
+        next unless File.file?(path)
 
-      scc_file = File.join(source_dir, SUSE::Connect::YaST::GLOBAL_CREDENTIALS_FILE)
-      copy_old_credentials_file(scc_file)
-    end
+        # check for the NCC credentials, we need to save them as the SCC credentials
+        new_path = if File.basename(path) == "NCCcredentials"
+          SUSE::Connect::YaST::GLOBAL_CREDENTIALS_FILE
+        else
+          File.join(dir, File.basename(path))
+        end
 
-    def self.copy_old_credentials_file(file)
-      return unless File.exist?(file)
+        copy_old_credentials_file(path, new_path)
+      end
+    end
 
-      new_file = SUSE::Connect::YaST::GLOBAL_CREDENTIALS_FILE
+    private_class_method def self.copy_old_credentials_file(file, new_file)
```

Comment:
it is used also here. Here it does not cause problem?

Reply:
No, because there is no yardoc comment... :confused:

Reply:
ok